src/scraper.rs

```diff
@@ -182,16 +183,27 @@ impl Scraper {
 
         dom.find_urls_as_strings()
             .into_iter()
-            .filter(|candidate| Scraper::should_visit(candidate, &url))
+            .filter(|candidate| Scraper::should_visit(candidate))
             .for_each(|next_url| {
-                let mut next_full_url = url.join(&next_url).unwrap();
+                let mut url_to_parse = next_url.clone();
+                // If url is `///tools.wmflabs.org/`, the app crashes
+                if url_to_parse.starts_with("///") { url_to_parse = url_to_parse.replacen("///", "//", 1) }
```

Comment:
I think this is part of the formatting fail haha. We just use `cargo fmt` for formatting, you can see how to install it [here](https://github.com/rust-lang/rustfmt#quick-start)

Reply:
I am using clion and the rust plugin, which uses cargo fmt (as far as I understand)